### ISSUE TITLE
How to find the best lr?

### ISSUE THREAD

**Issue by fermat97** [davidtvs/pytorch-lr-finder#56]
Hi, I am new to this package. I have done the things [here](https://github.com/davidtvs/pytorch-lr-finder#leslie-smiths-approach). It gives me set of loss functions with their corresponding `lr`. But I don't know how to find the best `lr`. I would like to know if there is a method to automatically find the proposed best `lr` within this package.

**Comment by davidtvs** [comment]
With the latest release, there's no way to get a learning rate suggestion. With the latest code from the master branch, the `plot` method will by default provide a suggestion. The suggested point is simply the point of steepest gradient in the curve.

However, in general, there's no single algorithm/method that will always provide a good suggestion. Half-way or two-thirds of the way on the descending part of the curve often give decent results. See #44 for more information.

**Comment by fermat97** [comment]
@davidtvs thanks for your explanation, I was actually looking for the suggestion which, as you said, is provided by `plot`.
